internal/resources/report_group.go

```diff
@@ -316,10 +352,15 @@ func (r *reportGroupResource) Schema(_ context.Context, _ resource.SchemaRequest
 						"first_match_only": schema.BoolAttribute{
 							Description: "Only report the first match.",
 							Optional:    true,
+							Computed:    true,
+							Default:     booldefault.StaticBool(false),
 						},
 						"recipients": schema.ListNestedAttribute{
 							Description: "Recipients for the notification.",
 							Optional:    true,
+							Validators: []validator.List{
+								listvalidator.SizeAtLeast(1),
```

Comment:
and jira's fine with an empty recipients list (although I thnk it does _accept_ them if they're there)